### PR TITLE
Updating Triage ROE Per additional Feedback

### PR DESCRIPTION
**Pull request by alexpappasoddball**
I've updated the doc based on a number of peoples feedback.  Let me know what you think, we still have another week to refine as needed.  Thanks!

### CODE REVIEW

File: Platform/Teams/Triage/Rules of Engagement with Triage.md [hunk 1]
```diff
@@ -22,13 +40,20 @@ NOTE: Assigning an issue to Triage using the [Triage Incident Template](https://
...
 
 When in doubt, submit any issue through our [Triage Incident Template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=triage%2C+triage-incident&template=triage-incident-template.md&title=) and the Triage team will ensure it gets through the process correctly!
 
+
+
+
+
+
+
+# I Don't think we need to define this anymore?
```
Comment (cvalarida) [inline Platform/Teams/Triage/Rules of Engagement with Triage.md]: :+1: I think you pretty much did that in-line, yeah.

File: Platform/Teams/Triage/Rules of Engagement with Triage.md [hunk 2]
```diff
@@ -40,13 +65,3 @@ When in doubt, submit any issue through our [Triage Incident Template](https://g
 
 Create a process flow diagram to compliment what is being said in a visaul manner
```
Comment (cvalarida) [inline Platform/Teams/Triage/Rules of Engagement with Triage.md]: Typo we missed earlier: `visaul` -> `visual`

**Review by cvalarida — APPROVED** [review]

File: Platform/Teams/Triage/Rules of Engagement with Triage.md [hunk 3]
```diff
@@ -1,18 +1,36 @@
...
+When the incident is believed to be **Severity 1** which can be defined as (but not limited to)
+ * Issue resolution is required in a 24 hour period
+ * Part of (or all of) va.gov is severely disrupted
   
-* When an issue is discovered that is not in your products scope of work
-  * Any issue that is discovered that is outside your teams scope of work can be reported to Triage
-  * In this case severity does not matter
+Examples:
+- va.gov is unresponsive
+- Platform errors causing extreme delays for users
```
Comment (omgitsbillryan) [inline Platform/Teams/Triage/Rules of Engagement with Triage.md]: Another good example: 
```
- Login to va.gov is unavailable
```

Comment (omgitsbillryan) [reply Platform/Teams/Triage/Rules of Engagement with Triage.md]: Also, I'm not sure that `- Platform errors causing extreme delays for users` is concrete enough. I'm a little fuzzy myself on exactly what that means.

**Review by omgitsbillryan — APPROVED** [review]
👍 only one small comment.

File: Platform/Teams/Triage/Rules of Engagement with Triage.md [hunk 4]
```diff
@@ -1,18 +1,36 @@
...
+  
+If you believe the issue falls into the Severity 1 category, there is no need to follow the Triage process.  Immediate notification to On Call is the best path to resolution.
+
+Slack - [#oncall](https://github.com/department-of-veterans-affairs/vets.gov-team/tree/master/Practice%20Areas/Engineering/OnCall)
+
+[Instructions on escalting to On Call can be found here](https://github.com/department-of-veterans-affairs/devops/blob/master/docs/Incident%20Response%20Playbook.md)
+
+## When to engage the Triage Team
+  
+**General Rule:** Any issue that is discovered that is outside your teams scope of work can be reported to Triage
```
Comment (omgitsbillryan) [inline Platform/Teams/Triage/Rules of Engagement with Triage.md]: We could provide examples here as well; similar to the examples for a sev 1 issue. We already have some real world examples 🙂 :

```
Examples:
- An internal load testing tool is broken
- Mock data is not working or is out of date
- Metrics are being reported incorrectly or not reported at all
```

File: Platform/Teams/Triage/Rules of Engagement with Triage.md [hunk 5]
```diff
@@ -1,18 +1,36 @@
 # Preface
 This document will help all team members working on VA.gov understand when it is appropriate to engage the Triage team for support and provide relevant information to assist in routing reported issues to other appropriate teams.  Because of the limited number of resources on Triage, they cannot be a "catch-all" for every issue reported on the platform and every application.  Please read the following document carefully in order to determine the appropriate path for your incident.
 
-## When it is OK to engage the Triage Team
+## Is this is a Severity 1 Issue? - Notify On Call Immediatley
```
Comment (kfrz) [inline Platform/Teams/Triage/Rules of Engagement with Triage.md]:
```suggestion
## Is this is a Severity 1 Issue? - Notify On Call Immediately
```

File: Platform/Teams/Triage/Rules of Engagement with Triage.md (hunk 3 above)
Comment (kfrz) [reply Platform/Teams/Triage/Rules of Engagement with Triage.md]: RE: Login being unavailable, would we want to discern from partial degradation, i.e. one of the providers is down (MHV) but not all?

File: Platform/Teams/Triage/Rules of Engagement with Triage.md [hunk 6]
```diff
@@ -1,18 +1,36 @@
...
+  
+**General Rule:** Any issue that is discovered that is outside your teams scope of work can be reported to Triage
+
+**How to Report the Issue:**
+Reach out on our [Slack Channel](https://dsva.slack.com/messages/CK1FA11H8)
+
+Submit a GitHub issue using our [Template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=triage%2C+triage-incident&template=triage-incident-template.md&title=)
+
+Is there already a GitHub Issue? Add the following tags and notify the Slack Channel for visibility:
+* Triage
```
Comment (kfrz) [inline Platform/Teams/Triage/Rules of Engagement with Triage.md]: `triage` `triage-incident` nitpick label formatting

File: Platform/Teams/Triage/Rules of Engagement with Triage.md (hunk 1 above)
Comment (kfrz) [reply Platform/Teams/Triage/Rules of Engagement with Triage.md]: that's a lot of extra whitespace

File: Platform/Teams/Triage/Rules of Engagement with Triage.md (hunk 2 above)
Comment (kfrz) [inline Platform/Teams/Triage/Rules of Engagement with Triage.md]:
```suggestion
Create a process flow diagram to compliment what is being said in a visual manner
```

Comment (kfrz) [reply Platform/Teams/Triage/Rules of Engagement with Triage.md]: @cvalarida I added a "suggested commit" to this

**Review by kfrz — APPROVED** [review]
Looks good overall, couple of nitpicks/typos

**Pull request merged**